subprojects/core/src/main/java/org/gradle/api/internal/project/DefaultProject.java

```diff
@@ -530,6 +532,14 @@ public Path getIdentityPath() {
     }
 
     @Override
+    public ConfigureProjectBuildOperationDescriptor getConfigureProjectBuildOperationDescriptor() {
+        if (configureProjectBuildOperationDescriptor == null) {
+            configureProjectBuildOperationDescriptor = new ConfigureProjectBuildOperationDescriptor(name, projectDir, getBuildFile(), path);
+        }
+        return configureProjectBuildOperationDescriptor;
```

Comment:
We should configure a project only once, so there should be no reason to retain this.